src/MenuItem.h

```diff
@@ -0,0 +1,39 @@
+#include <stdlib.h>
+#include <fstream>
+#include <iostream>
+
+#include <boost/filesystem/operations.hpp>
+#include <boost/filesystem/fstream.hpp>
+#include <boost/filesystem.hpp>
+
+#include <Wt/WContainerWidget>
+#include <Wt/WHBoxLayout>
+#include <Wt/WVBoxLayout>
+#include <Wt/WPushButton>
+#include <Wt/WCheckBox>
+#include <Wt/WSelectionBox>
+#include <Wt/WApplication>
+#include <Wt/WBreak>
+#include <Wt/WMenuItem>
+#include <Wt/WPopupMenu>
```

Comment:
again, you are doing the same. copy headers include from existing file. Please please do include only necessary. It had warned before